CriticalMass/AppDataStore.swift

```diff
@@ -0,0 +1,149 @@
+//
+//  AppDataStore.swift
+//  CriticalMaps
+//
+//  Created by Leonard Thomas on 1/15/19.
+//
+
+import CoreData
+import UIKit
+
+public class AppDataStore: DataStore {
+    private var userDefaults: UserDefaults
+
+    public init(userDefaults: UserDefaults = .standard) {
+        self.userDefaults = userDefaults
+        loadFriends()
```

Comment:
We should do a FeatureCheck here otherwise we would create an SQLLite Database in production. We shouldn't do that because there might be spec changes before we ship the feature which might require database migrations